ra/ra_test.go

```diff
@@ -491,8 +491,8 @@ func TestNewAuthorization(t *testing.T) {
 	test.Assert(t, len(authz.Challenges) == len(SupportedChallenges), "Incorrect number of challenges returned")
 	test.Assert(t, SupportedChallenges[authz.Challenges[0].Type], fmt.Sprintf("Unsupported challenge: %s", authz.Challenges[0].Type))
 	test.Assert(t, SupportedChallenges[authz.Challenges[1].Type], fmt.Sprintf("Unsupported challenge: %s", authz.Challenges[1].Type))
-	test.Assert(t, authz.Challenges[0].IsSane(false), "Challenge 0 is not sane")
-	test.Assert(t, authz.Challenges[1].IsSane(false), "Challenge 1 is not sane")
+	test.AssertNotError(t, authz.Challenges[0].CheckConsistencyForClientOffer(), "Challenge 0 is not sane")
+	test.AssertNotError(t, authz.Challenges[1].CheckConsistencyForClientOffer(), "Challenge 1 is not sane")
```

Comment:
Can you update these two assert messages to use the new terminology instead of "not sane"?